src/app/components/shared/audio-event-card/annotation-event-card.component.scss

```diff
@@ -11,13 +11,30 @@
 oe-axes {
   display: block;
   width: 332px;
+
+  // I am trialing using content-visibility so that spectrograms down the page
+  // don't have to paint until the user starts to scroll.
+  // This should improve the initial page load and responsiveness.
+  // However, it does have the possibility to create lag when scrolling because
+  // content will be painted on scroll.
+  //
+  // To see what this does, enable "show paints" in the Chrome dev tools
+  // "layers" tab.
+  //
+  // If this proves to be bad for UX, we should revert this change.
+  content-visibility: auto;
```

Comment:
does this also delay the network request for the audio? how much of spectrogram isn't rendered? is the element made and just not painted?

Reply:
Here's a small demo showing what the browser is painting when scrolling the page

https://github.com/user-attachments/assets/aead8928-b614-4468-bf2b-24c632c70925

Note that any lag seen is because I have the Chrome dev tools "paint" visualization enabled

Reply:
ok so it looks great, but i'd typically like to see an experiment done with some kind of quantifiable measurement of effect.

e.g. without this setting, page load and idle time was 3.5s, and with it was 3.3s, (10 runs averaged).

I'm not going to insist on this, but it's the only way to deal with perf improvements over a long period of time